samples/Constraint/data_constraint_cuberegion.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<mes:Structure xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xml="http://www.w3.org/XML/1998/namespace" xmlns:mes="http://www.sdmx.org/resources/sdmxml/schemas/v3_1_0/message" xmlns:str="http://www.sdmx.org/resources/sdmxml/schemas/v3_1_0/structure" xmlns:com="http://www.sdmx.org/resources/sdmxml/schemas/v3_1_0/common" xsi:schemaLocation="http://www.sdmx.org/resources/sdmxml/schemas/v3_1_0/message ../../schemas/SDMXMessage.xsd">
+	<mes:Header>
+		<mes:ID>IREF583870</mes:ID>
+		<mes:Test>false</mes:Test>
+		<mes:Prepared>2024-07-30T16:53:05Z</mes:Prepared>
+		<mes:Sender id="Unknown"/>
+		<mes:Receiver id="not_supplied"/>
+	</mes:Header>
+	<mes:Structures>
+		<str:DataConstraints>
+			<str:DataConstraint urn="urn:sdmx:org.sdmx.infomodel.registry.DataConstraint=ECB:EXR_CONSTRAINTS(1.0)" isExternalReference="false" agencyID="ECB" id="EXR_CONSTRAINTS" version="1.0">
+				<com:Name xml:lang="en">Allowed data constraints for the EXR dataflow.</com:Name>
+				<str:CubeRegion>
+					<str:Component id="CURRENCY_DENOM">

```

Comment:
Finally, `Keyvalue` was not replaced by `Component` thus this should be reverted.

Reply:
Are you asking for the schema to change and the example, i.e. to revert both to KeyValue?